pkg/ioutils/readers_test.go

```diff
@@ -45,7 +45,7 @@ func TestReaderErrWrapperReadOnError(t *testing.T) {
 func TestReaderErrWrapperRead(t *testing.T) {
 	reader := strings.NewReader("a string reader.")
 	wrapper := NewReaderErrWrapper(reader, func() {
-		t.Fatalf("readErrWrapper should not have called the anonymous function on failure")
+		t.Fatalf("readCloserWrapper should not be called")
```

Comment:
The comment is copied from previous test, actually it is not an error and the anonymous func is not going to be called. Do you want me to change the wording?